### PR TITLE
[stdlib]Replace NSMutableSet with Set

### PR DESCRIPTION
**Pull request by Moonisky**
1.I think replace the `NSMutableSet` with `Set<UIView>`(and `Set<NSView>`)
could make the operations more Swift-like.

These code works fine in my Playground, and I also pass the test. But
I’m not quite sure it’s ok to use `Set` here rather than `NSMutableSet`.

2.I also simplify the read-only computed property in “UIKit.swift”

### CODE REVIEW

File: stdlib/public/SDK/AppKit/AppKit.swift [hunk 1]
```diff
@@ -78,9 +78,8 @@ struct _NSViewMirror : _MirrorType {
       // This code checks that we aren't trying to log the same view recursively - and if so just returns
       // nil, which is probably a safer option than crashing
       // FIXME: is there a way to say "cacheDisplayInRect butDoNotRedrawEvenIfISaidSo"?
-      switch _NSViewMirror._views.member(_v) {
-        case nil:
-          _NSViewMirror._views.addObject(_v)
+      if !_NSViewMirror._views.contains(_v) {
+          _NSViewMirror._views.insert(_v)
```
Comment (gribozavr) [inline stdlib/public/SDK/AppKit/AppKit.swift]: Could you fix up the indentation after removing one level of nesting?  We use two spaces.

File: stdlib/public/SDK/UIKit/UIKit.swift [hunk 2]
```diff
@@ -47,19 +47,18 @@ extension UIOffset : Equatable {}
...
   }
 
   var isFlat: Bool {
-    get { return self == .FaceUp || self == .FaceDown }
+    return self == .FaceUp || self == .FaceDown
   }
 
   var isValidInterfaceOrientation: Bool {
-    get {
       switch (self) {
```
Comment (gribozavr) [inline stdlib/public/SDK/UIKit/UIKit.swift]: And another indentation fix here.

File: stdlib/public/SDK/UIKit/UIKit.swift [hunk 3]
```diff
@@ -169,34 +167,33 @@ public extension UIAlertView {
...
   var quickLookObject: PlaygroundQuickLook? {
       // iOS 7 or greater only
       
       var result: PlaygroundQuickLook? = nil
       
-      switch _UIViewMirror._views.member(_v) {
-        case nil:
-          _UIViewMirror._views.addObject(_v)
+      if !_UIViewMirror._views.contains(_v) {
+          _UIViewMirror._views.insert(_v)
```
Comment (gribozavr) [inline stdlib/public/SDK/UIKit/UIKit.swift]: And here.

File: stdlib/public/SDK/AppKit/AppKit.swift (hunk 1 above)
Comment (SemperIdem) [inline stdlib/public/SDK/AppKit/AppKit.swift]: OK, I forget to follow your guidelines, my mistake~:)

**Comment by SemperIdem** [comment]
@gribozavr I have fixed these indentation issues.

**Comment by gribozavr** [comment]
@SemperIdem Thank you!

**Comment by SemperIdem** [comment]
@gribozavr it's my pleasure. ヽ(･ω･｡)ﾉ

**Pull request merged**